server/file_hooks.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/plugin"
+)
+
+// FileWillBeUploaded is invoked after a file is uploaded.
```

Comment:
```suggestion
// FileWillBeUploaded is invoked when a file is uploaded, but before it is committed to backing store
```

See https://developers.mattermost.com/extend/plugins/server/reference/#Hooks.FileWillBeUploaded

Reply:
Fixed.